### DISCUSSION TITLE
Change the appearance of the cursor when interactions are possible

### DISCUSSION THREAD

**Discussion by EVAST9919** [ppy/osu#13038] [Ideas]
In some cases it's not too obvious what some UI elements can be interactable.
Usage:
- Draggable elements
- Resizable elements
- Rotatable elements (in the future)

Examples:
![1](https://user-images.githubusercontent.com/22874522/61476941-cf52a800-a996-11e9-85f6-15db22637520.png)
![2](https://user-images.githubusercontent.com/22874522/61476942-cf52a800-a996-11e9-9ed9-d2166d772b59.png)
![3](https://user-images.githubusercontent.com/22874522/61476943-cfeb3e80-a996-11e9-85ec-2153f758b971.png)

**Comment by peppy** [comment]
As we are targeting touch-screen devices, I'd hope no UX logic is attached to the cursor directly (as soon as we have the ability, the cursor will be hidden in such scenarios). I'd hope we can design the UI to be self-explanatory instead.
